src/plugins/auto/plugin.cpp

```diff
@@ -528,6 +548,42 @@ IExecutableNetworkInternal::Ptr MultiDeviceInferencePlugin::LoadNetworkImpl(cons
                                      1 /*single thread per stream*/,
                                      IStreamsExecutor::ThreadBindingType::NONE});
     executor->runAndWait(loads);
+
+    if (iterCPU != metaDevices.end()) {
+        int nCPUCores = parallel_get_max_threads();
```

Comment:
What is the replacement if OneTBB/TBB is not used? Check the MACRO for TBB/OneTBB

Reply:
use getNumberOfCPUCores() to get CPU Cores